src/classes/PS_ElevateService.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>48.0</apiVersion>
```

Comment:
@force2b , should we be using API version 49 for new classes/LWCs?

Reply:
Yeah. I think we should